css/projects.css

```diff
@@ -120,60 +128,93 @@ body {
 
 .project {
   position: relative;
+  box-sizing: border-box;
```

Comment:
So I wasn't sure about browser compatibility with `box-sizing` so I tested this in Safari and it only populates one column
![screen shot 2016-08-03 at 17 33 10](https://cloud.githubusercontent.com/assets/617994/17382746/70f062c8-59a0-11e6-956d-458e4459aae1.png)

However, it also does this in the current pulse:
![screen shot 2016-08-03 at 17 34 12](https://cloud.githubusercontent.com/assets/617994/17382762/83ae3c78-59a0-11e6-9f81-c1912b26a1b1.png)

so it's probably a separate issue from this PR. I'll file a separate issue for this.

Safari Version 9.1.2 (9537.86.7.7) on OSX 10.9.5